polyfill/lib/ecmascript.mjs

```diff
@@ -2014,18 +2014,19 @@ export function CalendarEquals(one, two) {
   const cal2 = ToTemporalCalendarIdentifier(two);
   return cal1 === cal2;
 }
+
 // This operation is not in the spec, it implements the following:
 // "If ? CalendarEquals(one, two) is false, throw a RangeError exception."
 // This is so that we can build an informative error message without
 // re-getting the .id properties.
-
-export function CalendarEqualsOrThrow(one, two, errorMessageAction) {
+export function ThrowIfCalendarsNotEqual(one, two, errorMessageAction) {
   if (one === two) return true;
```

Comment:
Based on your comment in https://github.com/tc39/proposal-temporal/commit/dcdc47f4505e24292c4eab6541a785fe631162ed, I agree that it makes more sense to change this `return true` to `return`

Reply:
Oops, I got distracted renaming the function and forgot to change the return value.

Fixed now.